<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Redact</class>
 <widget class="QDialog" name="Redact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(48, 48, 222);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Фамилия</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Имя</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отчество</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Возраст</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Населенный пункт</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(48, 48, 222);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
